src/main/java/at/hannibal2/skyhanni/config/features/chat/ChatConfig.java

```diff
@@ -108,36 +109,36 @@ public String toString() {
     @Expose
     @ConfigOption(
         name = "Translator",
-        desc = "Click on a message to translate it to English. " +
-            "Use §e/shcopytranslation§7 to translate from English. " +
+        desc = "Click on a message to translate it to English.\n" +
+            "Use §e/shcopytranslation§7 to translate from English.\n" +
             "§cTranslation is not guaranteed to be 100% accurate."
     )
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean translator = false;
 
     @Expose
-    @ConfigOption(name = "Skyblock XP Chat", desc = "Sends the Skyblock XP message into the chat.")
+    @ConfigOption(name = "Skyblock XP in Chat", desc = "Send the Skyblock XP messages into the chat.")
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean skyBlockXPInChat = true;
 
     @Expose
-    @ConfigOption(name = "Jacob's Event", desc = "Hide Anita Accessories' fortune bonus messages outside the Garden.")
+    @ConfigOption(name = "Anita's Accessory Messages", desc = "Hide Anita's Accessories' fortune bonus messages outside the Garden.")
```

Comment:
remove the word "message" in the title. try to keep the title as short as possible to avoid moulconfig (our config engine) making the title smaller. (same above)